src/index.ts

```diff
@@ -69,12 +69,15 @@ export interface IClientConfig {
 
 export { IAssignmentLogger, IAssignmentEvent, IEppoClient } from '@eppo/js-client-sdk-common';
 
+const localStorage = new EppoLocalStorage();
```

Comment:
Moved this back to the module-level (where it was) so it can be used in the static initialization of `EppoJSClient.instance`